solc/CommandLineInterface.cpp

```diff
@@ -416,8 +418,13 @@ void CommandLineInterface::readInputFilesAndConfigureRemappings()
 				auto infile = boost::filesystem::path(path);
 				if (!boost::filesystem::exists(infile))
 				{
-					cerr << "Skipping non-existent input file \"" << infile << "\"" << endl;
```

Comment:
Just turn this into an error. It should not be an option that you can switch on or off.

Reply:
@chriseth we want to silently ignore the missing file?  Is there a reason we wouldn't to simply inform the user we are skipping it (for education, debugging, general UX)?  Not taking stance here, simply seeking to better understand your POV.  Thanks for all the feedback so far.

Reply:
No, just stop processing. I'm not sure we need an option to allow files to be missing.